server/workflow/impl/Packrat/WorkflowUpload.ts

```diff
@@ -114,6 +114,8 @@ export class WorkflowUpload implements WF.IWorkflow {
                     return this.handleError(`WorkflowUpload.validateFiles unable to unzip asset version ${RSR.fileName}: ${zipRes.error}`);
 
                 const files: string[] = await ZS.getJustFiles(null);
+                if (!files || files.length === 0)
+                    return this.handleError(`WorkflowUpload.validateFiles unable to detect files in zip of asset version ${JSON.stringify(assetVersion, H.Helpers.saferStringify)}`);

```

Comment:
I believe that this error message is displayed to the user in the client.  If so, we don't want to add the stringified asset version.  Use a message like "zip file is unexpectedly empty".